josemak25/baby-bliss
# Fix flatlist for better render and optimised card size for all screen sizes

#### What does this PR do?

Have the app home page display all post and blogs and have each post item responsive to the screen size.

#### Description of Task to be completed?

Have the get started screen show the following:

1. Display all post on the home page.
2. Show an animated loader when the image is loading.
3. On scroll make he flatlist show item in a fast way
4. and display blog cards in a grid form

#### How should this be manually tested?

After cloning the repo,
`CD` into it and `RUN`

```javascript
git checkout fix-optimizing-flatlist

git pull origin fix-optimizing-flatlist

yarn install

yarn test

```
Using an emulator: Launch the app, login and you should be on the home page with the post displayed as dummy data

#### Screenshots (if appropriate)

#### Iphone screen of 375
![Simulator Screen Shot - iPhone 11 - 2020-03-02 at 09 25 30](https://user-images.githubusercontent.com/33870502/75658086-e3b68d00-5c67-11ea-9775-537da5d32fc3.png)

#### Iphone screen of 414
![Simulator Screen Shot - iPhone 7 - 2020-03-02 at 09 25 28](https://user-images.githubusercontent.com/33870502/75658120-f4670300-5c67-11ea-8139-3af3ae5bfd7a.png)

#### Ipads
![Simulator Screen Shot - iPad Air (3rd generation) - 2020-03-02 at 09 25 29](https://user-images.githubusercontent.com/33870502/75659419-414bd900-5c6a-11ea-8dd5-f40b6a81942e.png)
